addresspoints/src/test/scala/grasshopper/addresspoints/protocol/AddressPointJsonProtocolSpec.scala

```diff
@@ -11,7 +11,7 @@ class AddressPointJsonProtocolSpec extends FlatSpec with MustMatchers with Addre
 
   "Status" should "convert to and from JSON" in {
     val date = Calendar.getInstance().getTime().toString
-    val status = Status("OK", "grasshopper-addresspoints", date, InetAddress.getLocalHost.getHostName)
+    val status = Status("OK", "grasshopper-grasshopper.addresspoints", date, InetAddress.getLocalHost.getHostName)
```

Comment:
Why the extra `grasshopper.` here?  It seems a unique/descriptive identifier as is.  And if you do want this format, what should grasshopper-parser set its service name to?  I assume it would be just "grasshopper-parser"?